code/src/layouts/header/components/header-toolbar/HeaderToolbar.tsx

```diff
@@ -100,14 +100,17 @@ const HeaderToolbar = () => {
 
   const loadingDashboardButton = isLoadingAuth && <AppLoader />;
 
-  const authenticatedDashboardButton = userRole === "ROLE_MANAGER" &&
-    !isLoadingAuth && (
-      <AppTooltip titleTranslationKey="dashboard.tooltip">
-        <AppIconButton to={routes.dashboard.path} component={AppLink}>
-          <DashboardCustomizeIcon fontSize="medium" />
-        </AppIconButton>
-      </AppTooltip>
-    );
+  const isDashboardAvailable =
+    (userRole === "ROLE_MANAGER" || userRole === "ROLE_ADMIN") &&
```

Comment:
Prefer using `ROLES.MANAGER` and `ROLES.ADMIN` here, because it will be easier to change roles only in constants file

Reply:
Thank you! Fixed